.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -13,8 +13,6 @@ body:
       options:
         - label: I have tried restarting my IDE and the issue persists.
           required: true
-        - label: I have pulled the latest `main` branch of the repository.
```

Comment:
Bugs are a problem with the distributed product.  People reporting bugs shouldn't have to pull the repository at all